agents/cli.py

```
@@ -0,0 +1,166 @@
+        callback = webhook_callbacks.get('task.create')
+        if callback:
+            result = await callback(payload)
+            return jsonify({
+                'status': 'success',
+                'result': result,
+                'timestamp': datetime.now().isoformat()
+            }), 201
+        
+        return jsonify({'error': 'No callback registered'}), 400
+    
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)